tests/integration_tests/test_sim.py

```diff
@@ -116,6 +118,17 @@ def test_pin_mode(self, qtbot, sim_app, device_type):
         assert hw.get_pin_mode(17) == PinModes.INPUT
         assert hw.get_pin_mode(34) == PinModes.OUTPUT
 
+    def test_voltage_reading(self, qtbot, sim_app, device_type):
+        voltage_sensor = VoltageSensor(36)
+        assert voltage_sensor.last_val == 0
+
+        hw = get_hw_sim(sim_app, device_type)
+        hw.replace_sensor(voltage_sensor)
+
+        time.sleep(1)
+
+        assert voltage_sensor.last_val == VoltageSensor.NOMINAL_VOLTAGE
```

Comment:
We should be reading the value from the data sent over radio and saved by the GS, not directly from the simulated sensor. That way we test the integration of the whole system.
```suggestion
        assertsim_app.rocket_data.last_value_by_device(device_type, DataEntryIds.VOLTAGE) == VoltageSensor.NOMINAL_VOLTAGE
```